controller/array_action/array_action_types.py

```diff
@@ -29,3 +31,16 @@ def __init__(self, name, volume_internal_id, other_volume_internal_id, copy_type
         self.copy_type = copy_type
         self.is_ready = is_ready
         self.is_primary = is_primary
+
+
+@dataclass
+class Host:
+    name: str
+    connectivity_types: list = field(repr=False)
+    nvme_nqns: list = field(default_factory=list, repr=False)
+    fc_wwns: list = field(default_factory=list, repr=False)
+    iscsi_iqns: list = field(default_factory=list, repr=False)
```

Comment:
shouldn't these also be private? or is it part of CSI-4250?

Reply:
yes, linked in the ticket to [this](https://github.com/IBM/ibm-block-csi-driver/pull/495#discussion_r862461563)

Reply:
the link is about the initiators, not (explicitly) the host.
could you add this conversation to the ticket as well?

Reply:
done